capstone/src/main/java/com/google/univiz/scorecard/CollegeDataApiImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.google.univiz.scorecard;
+
+import static java.util.stream.Collectors.toList;
+
+import com.google.gson.Gson;
+import com.google.univiz.CollegeData;
+import com.google.univiz.CollegeDataApi;
+import com.google.univiz.api.CollegeId;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import javax.inject.Inject;
+
+final class CollegeDataApiImpl implements CollegeDataApi {
+  private final ScorecardConverter scorecardConverter;
+  private final CollegeIdReaderProvider readerProvider;
+  private final URLProvider urlProvider;
+  private final Gson gson;
+
+  @Inject
+  protected CollegeDataApiImpl(
```

Comment:
No need to make it protected. The class is final so it cannot have sub-classes.
```suggestion
  CollegeDataApiImpl(
```